index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="normalize.css" />
+    <link rel="stylesheet" href="style.css" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Apple Clone</title>
+  </head>
+  <body>
+    <nav class="bg-image">
+      <div class="nav-flex-container">
+        <div class="flex-items item1">
+          <img src="./images/apple_logo.gif" alt="" /> <span></span>
+        </div>
+        <div class="flex-items item2">Store</div>
+        <div class="flex-items item3">Mac</div>
+        <div class="flex-items item4">iPod</div>
+        <div class="flex-items item5">iPhone</div>
+        <div class="flex-items item6">iPad</div>
+        <div class="flex-items item7">iTunes</div>
+        <div class="flex-items item8">Support</div>
+        <div class="flex-items item9">
+          <span></span>
```

Comment:
**STATUS: REQUIRED CHANGES
Hello @e71az,**
It looks almost perfect to me and most of the requirements are implemented but there are a couple of minor changes required. Let's fix them.
[Required] we have to add a link to each of the nav-items. [Link to the page](https://web.archive.org/web/20140301004610/http://www.apple.com/)